spec/lib/kiev/request_body_filter_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "rack"
+
+describe Kiev::RequestBodyFilter do
+  describe ".filter" do
+    let(:request) do
+      Rack::Request.new(
+        Rack::MockRequest.env_for(
+          "/",
+          "REQUEST_METHOD" => "POST",
+          "CONTENT_TYPE" => "application/json",
+          :input => "{\"password\":12345}"
+        )
+      )
+    end
+
+    it "doesn't filter" do
+      initial = Kiev.config.disable_filter_for_log_levels
```

Comment:
`before` ?

Reply:
There is only one spec as far as I remember, so...what for? :)